docs/resources/aws_ec2_client_vpn_route.md

```diff
@@ -5,82 +5,82 @@ platform: aws
 
 # aws_ec2_client_vpn_route
 
-Use the `aws_ec2_client_vpn_route` InSpec audit resource to test properties of a single specific AWS EC2 Client VPN route.
+Use the `aws_ec2_client_vpn_route` InSpec audit resource to test the properties of a single specific AWS EC2 client VPN route.
 
-The `AWS::EC2::ClientVpnRoute` specifies a network route to add to a Client VPN endpoint.
+The `AWS::EC2::ClientVpnRoute` specifies a network route to add to a client VPN endpoint.
```

Comment:
This is super annoying and one of the reasons that software companies should name products after generic terms, but "Client VPN" is an [AWS product](https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/what-is.html), so it should be capitalized.